### PR TITLE
Nisha | Test | bug fixes Item wise export report

### PR DESCRIPTION
**Pull request by nisha-walkover**
https://app.clickup.com/t/86cvrkee7

### CODE REVIEW

File: apps/web-giddh/src/app/downloads/components/exports/exports.component.ts [hunk 1]
```diff
@@ -112,8 +112,22 @@ export class ExportsComponent implements OnInit, OnDestroy {
      * @memberof ExportsComponent
      */
     public openDialog(row: any): void {
+        const exportType = [
+            'INVENTORY_ITEM_WISE_EXPORT',
+            'INVENTORY_VARIANT_WISE_EXPORT',
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/downloads/components/exports/exports.component.ts]: move to enum file and make it public variable

File: apps/web-giddh/src/app/downloads/components/exports/exports.component.ts [hunk 2]
```diff
@@ -112,8 +112,22 @@ export class ExportsComponent implements OnInit, OnDestroy {
...
+            'INVENTORY_GROUP_WISE_EXPORT',
+            'INVENTORY_TRANSACTION_WISE_EXPORT'
+        ];
+        let dataToPass = row;
+        if (exportType.includes(row?.type)) {
+            dataToPass = row?.inventoryReportsExportFilter;
+            console.log(row?.inventoryReportsExportFilter);
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/downloads/components/exports/exports.component.ts]: remove console

File: apps/web-giddh/src/app/downloads/components/exports/exports.component.ts [hunk 3]
```diff
@@ -112,8 +112,22 @@ export class ExportsComponent implements OnInit, OnDestroy {
      * @memberof ExportsComponent
      */
     public openDialog(row: any): void {
+        const exportType = [
+            'INVENTORY_ITEM_WISE_EXPORT',
+            'INVENTORY_VARIANT_WISE_EXPORT',
+            'INVENTORY_GROUP_WISE_EXPORT',
+            'INVENTORY_TRANSACTION_WISE_EXPORT'
+        ];
+        let dataToPass = row;
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/downloads/components/exports/exports.component.ts]:
```suggestion
        let dataReq ;
```

File: apps/web-giddh/src/app/downloads/components/exports/exports.component.ts [hunk 4]
```diff
@@ -112,8 +112,22 @@ export class ExportsComponent implements OnInit, OnDestroy {
...
+        if (exportType.includes(row?.type)) {
+            dataToPass = row?.inventoryReportsExportFilter;
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/downloads/components/exports/exports.component.ts]:
```suggestion
            dataReq = row?.inventoryReportsExportFilter;
```

File: apps/web-giddh/src/app/new-inventory/component/report-filters/report-filters.component.html [hunk 5]
```diff
@@ -192,6 +195,7 @@
                 [iconClass]="'icon-options mr-2'"
                 [cssClass]="'customise-filter-button'"
                 [buttonText]="localeData?.header?.customise"
+                style="width: 33%;"
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/new-inventory/component/report-filters/report-filters.component.html]: remove inline css

File: apps/web-giddh/src/app/new-inventory/component/report-filters/report-filters.component.ts [hunk 6]
```diff
@@ -308,6 +313,59 @@ export class ReportFiltersComponent implements OnInit, OnChanges, OnDestroy {
...
+             showGroupName: false,
+             showUnitName: false,
+             showOpeningStockQty: false,
+             showOpeningStockValue: false,
+             showInwardsQty: false,
+             showInwardsValue: false,
+             showOutwardsQty: false,
+             showOutwardsValue: false,
+             showClosingStockQty: false,
+             showClosingStockValue: false,
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/new-inventory/component/report-filters/report-filters.component.ts]:
```suggestion
             showClosingStockValue: false
```

File: apps/web-giddh/src/app/new-inventory/component/report-filters/report-filters.component.ts [hunk 7]
```diff
@@ -308,6 +313,59 @@ export class ReportFiltersComponent implements OnInit, OnChanges, OnDestroy {
...
+         }
+
+        /* for column value filter selected */
+         this.displayedColumns.forEach(column => {
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/new-inventory/component/report-filters/report-filters.component.ts]:
```suggestion
         this.displayedColumns?.forEach(column => {
```

File: apps/web-giddh/src/app/new-inventory/component/report-filters/report-filters.component.ts [hunk 8]
```diff
@@ -426,6 +484,7 @@ export class ReportFiltersComponent implements OnInit, OnChanges, OnDestroy {
             if (!this.isCompany) {
                 this.stockReportRequest.branchUniqueNames = [this.generalService.currentBranchUniqueName];
                 this.balanceStockReportRequest.branchUniqueNames = [this.generalService.currentBranchUniqueName];
+                this.stockReportRequestExport.branchUniqueNames = [this.generalService.currentBranchUniqueName];
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/new-inventory/component/report-filters/report-filters.component.ts]:
```suggestion
                this.stockReportRequestExport.branchUniqueNames = this.generalService.currentBranchUniqueName ? [this.generalService.currentBranchUniqueName] : [];
```

**Review by dilpreetsinghofficial — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/downloads/components/exports/exports.component.ts [hunk 9]
```diff
@@ -79,6 +80,13 @@ export class ExportsComponent implements OnInit, OnDestroy {
     public fromDate: string;
     /** Instance of is electron variable */
     public isElectron: any = isElectron;
+    /** enum for export data in inventory */
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/downloads/components/exports/exports.component.ts]:
```suggestion
    /** Instance for export data in inventory */
```

**Review by dilpreetsinghofficial — APPROVED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**